Composer/packages/client/src/pages/botProject/adapters/ABSChannels.tsx

```diff
@@ -491,29 +495,32 @@ export const ABSChannels: React.FC<RuntimeSettingsProps> = (props) => {
           <Spinner />
         </Stack.Item>
       )}
-      {key === CHANNELS.TEAMS && channelStatus?.[key].enabled && !channelStatus?.[key].loading && (
-        <Stack.Item>
-          <Link
-            styles={{ root: { marginTop: '7px' } }}
-            onClick={() => {
-              setShowTeamsManifestModal(true);
-            }}
-          >
-            {formatMessage('Open Manifest')}
-          </Link>
-        </Stack.Item>
-      )}
     </Stack>
   );
 
   const absTableRow = (channel: string, name: string, link: string) => (
     <div key={channel} css={tableRow}>
       <div css={tableRowItem(columnSizes[0])}>{name}</div>
       <div css={tableRowItem(columnSizes[1])}>{absTableToggle(channel)}</div>
-      <div css={tableRowItem(columnSizes[2])}>
-        <Link href={link} target="_docs">
-          {formatMessage('Learn more')}
-        </Link>
+      <div css={tableRowItem(columnSizes[0])}>
```

Comment:
If you're going to alter column widths, I'd prefer you either change the constant in `styles.ts` or make a new one - the `columnSizes` array there has three elements to match the three columns of the table, for an easy one-to-one correspondence.

Reply:
handled